src/databricks/labs/ucx/install.py

```diff
@@ -893,13 +895,28 @@ def repair_run(self, workflow):
                 return
             latest_job_run = job_runs[0]
             state = latest_job_run.state
+
+            while not state.result_state and (time.time() - start_time < timeout):
+                logger.info("Waiting for the result_state to update the state")
+                time.sleep(10)
```

Comment:
This is not unit testable, see how we use retried() decorator in workspace access package (dbsql permissions, secrets acls, etc).

Reply:
@nfx .Updated the code with retried logic.